packages/loopring_v3/contracts/core/impl/libtransactions/WithdrawTransaction.sol

```diff
@@ -64,7 +66,7 @@ library WithdrawTransaction
         uint    minGas;
         address to;
         bytes   extraData;
-        uint96  maxFee;
+        uint16  feeDiscount;
```

Comment:
I think `minGas` and `gasLimit` can also use uint24, the ethereum block's max gas limit is ~10,000,000. uint24's max value is 16777216.